packages/create-block/lib/templates/plugin/README.md.mustache

```diff
@@ -0,0 +1,114 @@
+# {{title}}
+
+{{title}} is a {{#isStaticVariant}}static{{/isStaticVariant}}{{#isDynamicVariant}}dynamic{{/isDynamicVariant}} block scaffolded by {{author}} using the `@wordpress/create-block` package.
+
+## The structure of the project
+```
+│
+│── build
+│
+│── node_modules
+│
+│── src
+│    │
+│    │── block.json
+│    │
+│    │── edit.js
+│    │
+│    │── editor.scss
+│    │
+│    │── index.js
+│    │
+{{#isDynamicVariant}}
+│    │── render.php
+{{/isDynamicVariant}}
+{{#isStaticVariant}}
+│    │── save.js
+{{/isStaticVariant}}
+│    │
+│    │── style.scss
+│    │
+│    └── view.js
+│
+│── .editorconfig
+│
+│── .gitignore
+│
+│── {{slugSnakeCase}}.php
+│
+│── package-lock.json
+│
+│── package.json
+│
+│── README.md (this file)
+│
+└── readme.txt
+```
+
+## Files and directories
+
+### build
+The `build` directory is where the final deployable build of the block will go. The build process will put compiled versions of files from the `src` directory in here. You should never need to touch this directory directly. 
+
+### node_modules
+The `node_modules` directory is where all the modules that the build process depends on live. Again you should never need to directly modify the contents of this directory. To install packages here use `npm install {package-name}` or define it as a 'dependency' or 'devDependency' in `package.json`.
+
+### src
+The `src` directory contains the files that you will primarily work with to create the markup and the functionality of your block. During the build step files here are compiled into the `build` directory. 
+
+#### src/block.json
+`block.json` stores the metadata for the block structured as a JSON object. Amongst other thing you’ll use this file to define attributes that the functions defined in edit.js and save.js can consume as parameters. [Learn more](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/)
+
+#### src/edit.js
+`edit.js` exports a component, `Edit()`, that is rendered in the editor and so determines how the block appears and functions within the editor. It is where you will define the markup and content of the block. It is also where you will provide controls for the user to interact with the block and customize the appearance and content of the block. The `Edit()` component defined in this file receives a props object which among other things will contain the attributes defined in `block.json`. `setAttributes` is also passed to allow you to update the attribute values.
+
+#### src/editor.scss
+`editor.scss` is a file containing SCSS that styles the appearance of the block in the block editor. It requires a compile step which will result in `build/index.css`. 
```

Comment:
SCSS is an extension. We should be clear that this file is CSS that will be compiled using Sass
```suggestion
`editor.scss` is a file containing CSS that will be compiled using [Sass](https://sass-lang.com/) that styles the appearance of the block in the block editor. It requires a compile step which will result in `build/index.css`. 
```